src/containers/add-course-banner/AddCourseBanner.styles.ts

```diff
@@ -0,0 +1,27 @@
+export const getStyles = (bannerUrl: string) => ({
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundSize: 'cover',
+    borderRadius: '10px 10px 0px 0px',
+    px: { md: 7, sm: 3, xs: 3 },
+    py: { md: 6, sm: 4, xs: 4 },
+    height: '216px',
+    backgroundImage: `linear-gradient(0deg, rgba(38, 50, 56, 0.15) 0%, rgba(38, 50, 56, 0.15) 100%), url(${bannerUrl})`,
+    '&:hover': {
+      backgroundImage: `linear-gradient(0deg, rgba(38, 50, 56, 0.40) 0%, rgba(38, 50, 56, 0.40) 100%), url(${bannerUrl})`
```

Comment:
can we add and use this colors in out `app.pallete.ts`?

Reply:
Gradients colors used only for this component, so i don't see real reasons for supplementing palette with them.
Maybe we'll add it later when these colors are used somewhere else